src/context/context.js

```diff
@@ -1,8 +1,10 @@
 import { createContext } from 'react';
+import LOCALES from '../i18n/locales';
```

Comment:
you don't need to add languages to the context

Reply:
Okay, removed